game/src/sandbox/dashboards/commuter.rs

```diff
@@ -379,7 +414,10 @@ fn make_panel(ctx: &mut EventCtx, app: &App) -> Composite {
             "Departing until:".draw_text(ctx).margin_right(20),
             AreaSlider::new(ctx, 0.25 * ctx.canvas.window_width, 1.0).named("depart until"),
         ]),
-        "Nothing selected".draw_text(ctx).named("current"),
+        "None selected"
+            .draw_text(ctx)
+            .named("current")
+            .margin_below(10),
```

Comment:
You don't need `margin_below` here; anything inside `Widget::col` gets this automatically.

I'll clean up all callers of `replace` and make them automatically steal the margin from the previous widget. That seems like a reasonable default, instead of every caller having to remember to put the margin back.